### PR TITLE
Refactor logrus setup and add TraceLevel

### PR DESCRIPTION
**Pull request by hramazani**

### CODE REVIEW

File: atlas/templates/cmd/server/main.go.gotmpl [hunk 1]
```diff
@@ -58,6 +58,7 @@ func NewLogger() *logrus.Logger {
 
 	// Set the log level on the default logger based on command line flag
 	logLevels := map[string]logrus.Level{
+		"trace":   logrus.TraceLevel,
```
Comment (kd7lxl) [inline atlas/templates/cmd/server/main.go.gotmpl:61]: A better solution would be to replace this whole section with https://pkg.go.dev/github.com/sirupsen/logrus#ParseLevel

**Review by kd7lxl — CHANGES_REQUESTED** [review]

File: atlas/templates/cmd/server/main.go.gotmpl (hunk 1 above)
Comment (hramazani) [reply atlas/templates/cmd/server/main.go.gotmpl]: done.

**Review by brocwoodworthIBLX — CHANGES_REQUESTED** [review]
looks good overall, please just fix the indentation - maybe run a `go fmt` on this file

**Comment by hramazani** [comment]
> looks good overall, please just fix the indentation - maybe run a `go fmt` on this file

Like ":set list!" in VIM, did enable show whitespaces in Goland to prevent this.
thanks.

Done.

**Review by kd7lxl — APPROVED** [review]

**Review by brocwoodworthIBLX — APPROVED** [review]

**Pull request merged**